cfgov/jinja2/v1/owning-a-home/explore-rates/index.html

```diff
@@ -392,7 +392,7 @@ <h4 class="rc-comparison-subhead">Interest costs over <span class="loan-years">3
 
           </div><!-- END .result -->
 
-          <section class="next-steps">
+          <section class="next-steps tabs-layout">
```

Comment:
Should we make this atomic?

Reply:
I wouldn't bother at this point. I think the whole tabs bit needs to be re-written. As is the tabs when in a non-JS state doesn't place the tab button text next to its content, but instead groups the tab button text and groups the content. If this goes into CF it'll get atomized and then can replace what's here.